### PR TITLE
LPS-187428 BrowserSniffer UpgradeProcess

### PR DESCRIPTION
**Pull request by alberttgomes**
[LPS-187428](https://liferay.atlassian.net/browse/LPS-187428)

### CODE REVIEW

**Comment by alberttgomes** [comment]
ci:test:sf

**Comment by alberttgomes** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/BaseTemplateUpgradeProcessTestCase.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+			upgradeProcess.upgrade();
+
+			_multiVMPool.clear();
+		}
+	}
+
+	@Inject
+	private DDMTemplate _ddmTemplate;
+
+	@Inject
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/BaseTemplateUpgradeProcessTestCase.java:128]: @alberttgomes, no need to have this `@Inject` annotation. It's used just to inject dependencies, like Liferay Services and Local Services, for example. Remember about it. :wink: Also, take some time to understand more about this annotation.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/build.gradle [hunk 2]
```diff
@@ -38,9 +38,12 @@ dependencies {
 	testIntegrationCompile project(":apps:data-engine:data-engine-rest-api")
 	testIntegrationCompile project(":apps:dynamic-data-mapping:dynamic-data-mapping-form-builder")
 	testIntegrationCompile project(":apps:export-import:export-import-test-util")
+	testIntegrationCompile project(":apps:fragment:fragment-api")
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/build.gradle:41]: @alberttgomes, please update commit message to be like:

Implement test to remove declared browser sniffer variables

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by alberttgomes** [comment]
@natocesarrego, resend on https://github.com/natocesarrego/liferay-portal/pull/1325

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/BaseTemplateUpgradeProcessTestCase.java (hunk 1 above)
Comment (alberttgomes) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/BaseTemplateUpgradeProcessTestCase.java:128]: please @natocesarrego !! Thx

**Pull request closed without merging**